charts/delphai-keda/templates/deployment.yml

```diff
@@ -35,6 +35,18 @@ spec:
       nodeSelector:
         node_type: "gpu"
       {{ end }}
+      affinity:
```

Comment:
Shouldn't this line also be under `if`?

Reply:
Yes, but eventually we can have other things under `affinity` other than `podAntiAffinity` right?

Reply:
Yes… but I'm not sure if that would work:
![image](https://user-images.githubusercontent.com/91895861/158964457-0f3f148a-fefd-44d1-95e6-cdeabfbe703c.png)

If `affinity` could be `null`. Did you test it?